src/Appium.Net/Appium/AppiumCommand.cs

```diff
@@ -211,13 +211,24 @@ public class AppiumCommand
 
             #endregion
 
+            #region Logs
+
             new AppiumCommand(HttpCommandInfo.GetCommand, DriverCommand.GetAvailableLogTypes,
                 "session/{sessionId}/log/types"),
-            new AppiumCommand(HttpCommandInfo.PostCommand, DriverCommand.GetLog, "session/{sessionId}/log")
+            new AppiumCommand(HttpCommandInfo.PostCommand, DriverCommand.GetLog, "session/{sessionId}/log"),
+
+            #endregion
+
+            #region Event logs
+
+            new AppiumCommand(HttpCommandInfo.PostCommand, AppiumDriverCommand.GetEvents, "session/{sessionId}/appium/events"),
+            new AppiumCommand(HttpCommandInfo.PostCommand, AppiumDriverCommand.LogEvent, "session/{sessionId}/appium/log_event")
+
+            #endregion
```

Comment:
Shouldn't it be placed after the closing bracket?

Reply:
Do you mean like this?

```
            #region Event logs

            new AppiumCommand(HttpCommandInfo.PostCommand, AppiumDriverCommand.GetEvents, "session/{sessionId}/appium/events"),
            new AppiumCommand(HttpCommandInfo.PostCommand, AppiumDriverCommand.LogEvent, "session/{sessionId}/appium/log_event")
        };

            #endregion
```

The beginning https://github.com/appium/dotnet-client/blob/7fe2acbac4fbc49fb6ab72b0933531a2fad645f8/src/Appium.Net/Appium/AppiumCommand.cs#L26 started in `{` so I thought the end also before `};`

Reply:
Ok, we can leave it as it is.
I was reviewing from my mobile, so I wasn't in front of the full code